plugins/module_utils/_hashi_vault_module.py

```diff
@@ -14,10 +14,7 @@
 
 from ansible.module_utils.basic import AnsibleModule
 
-from ansible_collections.community.hashi_vault.plugins.module_utils._hashi_vault_common import (
-    HashiVaultHelper,
-    HashiVaultOptionAdapter,
-)
+from ansible_collections.community.hashi_vault.plugins.module_utils._hashi_vault_common import HashiVaultHelper, HashiVaultHVACError, HashiVaultOptionAdapter
```

Comment:
nit: please revert back to the multi-line style for this, it makes it easier to see when the list was changed in git history